dags/analyzer_helper/common/analyzer.py

```diff
@@ -16,7 +16,7 @@ def analyze(
         action: dict[str, int],
         window: dict[str, int],
         recompute: bool,
-        config: PlatformConfigBase = DiscordAnalyzerConfig(),
+        config: PlatformConfigBase = TelegramAnalyzerConfig(),
```

Comment:
Please revert this part. The telegram config is still not written and also we'll remove the default config value in future. I'll handle the config after your PR being merged